vita3k/modules/SceAppUtil/SceAppUtil.cpp

```diff
@@ -21,8 +21,10 @@
 #include <psp2/system_param.h>
 
 #include <host/app_util.h>
+
```

Comment:
delete newline

Reply:
Oh, i have just add line or séparateur groupe like other

Reply:
no. that's not our convention

Reply:
^